# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1274781024 -7200
#      Tue May 25 11:50:24 2010 +0200
# Node ID ca836bcdf85ef103de064dce470135ab070de156
# Parent  9e4407c19e8d16706ad2579e9b45c72983a50865
don't mention onCleanup an missing

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-05-25  Jaroslav Hajek  <highegg@gmail.com>
+
+	* miscellaneous/unimplemented.m: Don't mention onCleanup (supported).
+
 2010-05-25  Jaroslav Hajek  <highegg@gmail.com>
 
 	* statistics/base/median.m: Ensure NaNs where needed. Simplify first
 	non-singleton dimension computation.
 
 2010-05-24  Rik <octave@nomad.inbox5.com>
         * general/private/__isequal__.m, image/image.m, 
         statistics/base/ranks.m: Replace calls to deprecated functions.
diff --git a/scripts/miscellaneous/unimplemented.m b/scripts/miscellaneous/unimplemented.m
--- a/scripts/miscellaneous/unimplemented.m
+++ b/scripts/miscellaneous/unimplemented.m
@@ -21,20 +21,16 @@
 ## @deftypefn {Function File} {} unimplemented ()
 ## Undocumented internal function.
 ## @end deftypefn
 
 function unimplemented (fcn)
 
   ## Some smarter cases, add more as needed.
   switch (fcn)
-  case "onCleanup"
-    txt = ["onCleanup is not implemented. Consider using unwind_protect, ",...
-    "which is more versatile. For more info, type @code{help unwind_protect}."];
-
   case "textscan"
     txt = ["textscan is not implemented. Consider using textread or sscanf."];
 
   case "gsvd"
     txt = ["gsvd is currently not part of Octave. See the linear-algebra package at ",...
     "@url{http://octave.sf.net/linear-algebra/}."];
 
   case "linprog"
@@ -291,17 +287,16 @@ function list = missing_functions ()
   "ode23s", 
   "ode23t", 
   "ode23tb", 
   "ode45", 
   "odefile", 
   "odeget", 
   "odeset", 
   "odextend", 
-  "onCleanup", 
   "open", 
   "openfig", 
   "opengl", 
   "openvar", 
   "ordeig", 
   "ordqz", 
   "ordschur", 
   "padecoef", 
